mlrun/runtimes/databricks_runtime_wrap.py

```diff
@@ -0,0 +1,173 @@
+                f.write(modified_code.encode("UTF8"))
+
+    workspace = WorkspaceClient(
+        token=mlrun.get_secret_or_env(key=token_key, secret_provider=get_secret_func)
+    )
+
+    now = datetime.datetime.now()
+    formatted_date_time = now.strftime("%Y-%m-%d_%H-%M-%S")
+    script_path_on_dbfs = (
+        f"/home/{workspace.current_user.me().user_name}/mlrun_databricks_run"
+        f"time/sample_{formatted_date_time}_{uuid.uuid4()}.py"
+    )
+
+    if is_local_code:
+        upload_file(
+            workspace=workspace,
+            script_path_on_dbfs=script_path_on_dbfs,
+            handler=internal_handler,
+        )
+
+    def print_status(run: Run):
+        statuses = [f"{t.task_key}: {t.state.life_cycle_state}" for t in run.tasks]
+        logger.info(f'workflow intermediate status: {", ".join(statuses)}')
+
+    try:
+        cluster_id = os.environ.get("DATABRICKS_CLUSTER_ID")
+        if cluster_id:
+            logger.info(f"run with exists cluster_id: {cluster_id}")
+            timeout = 15
+            waiter = workspace.jobs.submit(
+                run_name=f"py-sdk-run-{formatted_date_time}",
+                tasks=[
+                    SubmitTask(
+                        task_key=f"hello_world-{formatted_date_time}",
+                        existing_cluster_id=cluster_id,
+                        spark_python_task=SparkPythonTask(
+                            python_file=f"dbfs:{script_path_on_dbfs}",
+                            parameters=[json.dumps(kwargs)],
+                        ),
+                    )
+                ],
+            )
+        else:
+            logger.info("run with new cluster_id")
+            timeout = 25
+            waiter = workspace.jobs.submit(
+                run_name=f"py-sdk-run-{formatted_date_time}",
+                tasks=[
+                    SubmitTask(
+                        task_key=f"hello_world-{formatted_date_time}",
+                        new_cluster=ClusterSpec(
+                            spark_version=workspace.clusters.select_spark_version(
+                                long_term_support=True
+                            ),
+                            node_type_id=workspace.clusters.select_node_type(
+                                local_disk=True
+                            ),
+                            num_workers=1,
+                        ),
+                        spark_python_task=SparkPythonTask(
+                            python_file=f"dbfs:{script_path_on_dbfs}",
+                            parameters=[json.dumps(kwargs)],
+                        ),
+                    )
+                ],
+            )
+        logger.info(f"starting to poll: {waiter.run_id}")
+        run = waiter.result(
+            timeout=datetime.timedelta(minutes=timeout), callback=print_status
+        )
+
+        run_output = workspace.jobs.get_run_output(run.tasks[0].run_id)
+        logger.info(f"run_output: {run_output}")
```

Comment:
Why not `context.log_result(<key>, run_output)`? This will put the result in the output for the runtime, which seems to be what you want here, isn't it?